src/utils/validation/registration/registrationValidation.ts

```diff
@@ -21,8 +21,6 @@ export const registrationValidationSchema = Yup.object().shape({
     npiSubjectHeading: Yup.string(),
     date: Yup.object().shape({
       year: Yup.number().required(ErrorMessage.REQUIRED),
-      month: Yup.number(),
-      day: Yup.number(),
```

Comment:
Litt usikker på dette, for nå vil det validere fint om `day="OST"`... Har du prøvd med `day: Yup.number().nullable()` og undefined eller null som verdi? 
Hvis det ikke går er det kanskje like greit å validere dette manuelt, med noe ala `Yup.test(value => !value || !isNan(value))` (husker ikke helt hvordan man gjør det)

Reply:
ja, har prøvd med `nullable()` og å sette day og month lik `undefined`. men ja - kan se på manuell validering!